ayyesu/Mind_Map-Frontend
# Theme

## Pull Request Description: Implement Dark and Light Theme Feature

### Summary:
This pull request introduces a new feature to support both dark and light themes in the application. Users can now switch between dark and light themes based on their preferences.

### Changes Made:
- Added functionality to toggle between dark and light themes.
- Implemented UI changes to reflect the selected theme, including background colors, text colors, and other theme-specific styles.
- Updated components to dynamically adjust their appearance based on the selected theme.
- Added configuration options for theme settings.

### How to Test:
1. Navigate to the settings section of the application.
2. Locate the theme settings option.
3. Toggle between the dark and light themes.
4. Observe the changes in the application's appearance, including background colors, text colors, and other UI elements.
5. Verify that the selected theme persists across sessions.

### Additional Notes:
- The implementation adheres to best practices and follows established design guidelines.
- Testing was performed to ensure compatibility across different browsers and screen sizes.
- Accessibility considerations were taken into account to ensure a positive user experience for all users.

### Checklist:
- [x] Code is formatted according to project guidelines.
- [x] All changes pass linting and other code quality checks.
